hack/debug.sh

```diff
@@ -73,7 +89,13 @@ do_master () {
     mkdir -p $logmaster
 
     # Log the generic system stuff
-    log_system $logmaster $aos_master_service
+    log_services=
+    for service in "${services[@]}"; do
+        if [ -n "${service}" ]; then
+            log_services="${log_services} ${service}"
+        fi
+    done
```

Comment:
I guess I didn't look closely at this code before... what it is even doing? ${services} should contain exactly three non-empty strings...